metadata-integration/java/datahub-client/src/main/java/datahub/client/kafka/KafkaEmitter.java

```diff
@@ -28,7 +28,7 @@
 @Slf4j
 public class KafkaEmitter implements Emitter {
 
-  public static final String DEFAULT_MCP_KAFKA_TOPIC = "MetadataChangeProposal_v1";
+  public static String DEFAULT_MCP_KAFKA_TOPIC = "MetadataChangeProposal_v1";
```

Comment:
Hey there - Looks like our linter really doesn't like this. Is there a better way to provide this topic? Perhaps a mechanism that users the constructor (but is also backwards compatible)